go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -647,6 +663,41 @@ func (s *VtctldServer) GetShard(ctx context.Context, req *vtctldatapb.GetShardRe
 	}, nil
 }
 
+// GetSrvKeyspaceNames is part of the vtctlservicepb.VtctldServer interface.
+func (s *VtctldServer) GetSrvKeyspaceNames(ctx context.Context, req *vtctldatapb.GetSrvKeyspaceNamesRequest) (*vtctldatapb.GetSrvKeyspaceNamesResponse, error) {
```

Comment:
This implementation seems to diverge from the existent one in vtctl: https://github.com/tinyspeck/vitess/blob/22bdd8a9304ec2499414a2526ea644a2925259d5/go/vt/vtctl/vtctl.go#L3240

Curious of the rationale for this

Reply:
In VTAdmin we've been finding that we almost always want to get "all of the things" for a given resource, so this let's us make one call to a vtctld rather than one call to get all the cells and then one call per cell.

Which reminds me ... I meant to update the old implementation to call this one!

Reply:
Just to demonstrate the old commands still work after my latest commit:

```
❯ vtctlclient -server "localhost:15999" GetSrvKeyspaceNames zone1
commerce
```